src/timetrackings/index.ts

```diff
@@ -1,36 +1,44 @@
 import BaseApi from '../baseApi';
+import { TimeTracking, StartTimeTrackingData, StopTimeTrackingData } from './types';
 import responseHandler from '../utils/responseHandlers';
-import { TimeTracking } from './types';
+import { AxiosRequestConfig } from 'axios';
+import { ReadOptions } from '../utils/types';
 
 export default class TimeTrackings extends BaseApi {
-  static resourceName = 'absences';
+  static resourceName = 'timeTrackings';
 
-  public read(): Promise<TimeTracking[]> {
-    const response = this.get<TimeTracking>(`${TimeTrackings.resourceName}/read`);
+  read(id: number, options: ReadOptions = {}) {
```

Comment:
Maybe a better name for options would be params, as options can't be set. I would also change the ReadOptions to ReadParams. And options/params should be optional. Also in readAll.